### ISSUE TITLE
Tooltip hiding

### ISSUE THREAD

**Issue by Disketaa** [null2264/CobbleGen#41]
Would it possible to make a config entry to hide the tooltips? I'm making vanilla-like modpack and I'm trying to hide all necessery information for me. It's not so important for me cause I'll put deepslate generation instruction in advancements.

Maybe something like this: (If you don't want to create second config file for mod configuration)

```
{
        "modSettings": { "tooltips": false },


	"cobbleGen": [
		{
			"id": "minecraft:cobblestone",
			"weight": 100.0,
			"minY": 0
		}
	]
}
```

**Comment by null2264** [comment]
Meta config is already planned (#39), I just haven't got the time to work on it, I'll add it to the to-do list for now...

**Comment by null2264** [comment]
Will be fixed on v5.2.0, sorry it took a while...
You'll see a new config file, `cobblegen-meta.json5`, just change "addTooltip" to false

**Comment by Disketaa** [comment]
Love it!
